section3/exercises/hashes.rb

```diff
@@ -61,41 +69,67 @@
 # Check out the following example of an array of Instagram posts:
 
 
-posts = ["image at beach", "holiday party", "adorable puppy", "video of cute baby"]
+#posts = ["image at beach", "holiday party", "adorable puppy", "video of cute baby"]
 
 
 # An Array of Hashes is probably more realistic. Hashes are a data type that's great
 # for storing more complex data. Below, we can store multiple key-value pairs
 # that relate to each individual Instagram post:
 
 
-posts = [
-  {
-    image_src: "./images/beach.png",
-    caption: "At the beach with my besties",
-    timestamp: "4:37 PM August 13, 2019",
-    number_likes: 0,
-    comments: []
-  }, 
-  {
-    image_src: "./images/holiday-party.png",
-    caption: "What a great holiday party omg",
-    timestamp: "11:37 PM December 31, 2019",
-    number_likes: 13,
-    comments: []
-  }
-]
-
-p posts
-p posts[0]
-
-
-# The code snippet above shows an Array with 2 elements. Each element in the Array is a 
-# Hash. Each of those Hashes has 4 key-value pairs. This may LOOK 
+#=begin
+#posts = [
+#  {
+#    image_src: "./images/beach.png",
+#   caption: "At the beach with my besties",
+#    timestamp: "4:37 PM August 13, 2019",
+#    number_likes: 0,
+#    comments: []
+#  },
+#  {
+#    image_src: "./images/holiday-party.png",
+#    caption: "What a great holiday party omg",
+#    timestamp: "11:37 PM December 31, 2019",
+#    number_likes: 13,
+#    comments: []
+#  }
+#]
+
+#p posts
+#p posts[0]
+
+
+# The code snippet above shows an Array with 2 elements. Each element in the Array is a
+# Hash. Each of those Hashes has 4 key-value pairs. This may LOOK
 # a bit daunting - it's OK! You don't need to be 100% comfortable with this, but it's
 # good to have some exposure before going into Mod 1.
 
 
-# YOU DO: Create an array of at least 3 EMAIL Hashes, using the same 
+# YOU DO: Create an array of at least 3 EMAIL Hashes, using the same
 # key-value pairs you used in your email Hash above.
 # Then, print the email Array to the terminal.
+
+gmail = [
+  {
+"From" => "pdthomson1239@gmail.com",
+"To" => "turing@gmail.com",
+"Subject" => "Tuition",
+"Date_sent" => "01/01/2022",
+"Time_sent" => "6:00 p.m"
+},
+{
```

Comment:
Use indents to make this easier to read! Since it's an array of objects, you want it to be super organized.